arch/rv32i/src/syscall.rs

```diff
@@ -12,21 +12,21 @@ use kernel::syscall::ContextSwitchReason;
 #[repr(C)]
 pub struct Riscv32iStoredState {
     /// Store all of the app registers.
-    regs: [usize; 31],
+    regs: [u32; 31],
```

Comment:
```suggestion
    regs: [usize; 31],
```

Reply:
Change done because of the reasons outlined in https://github.com/tock/tock/pull/2446#discussion_r580291135. Will change if that can be resolved.